charts/fybrik/templates/manager-deployment.yaml

```diff
@@ -96,6 +96,20 @@ spec:
                 name: razee-credentials
             {{- end }}
           env:
+            - name: TLS_ENABLED
+              value: {{ .Values.global.tls.enabled | quote | toString }}
+            {{- if .Values.global.tls.enabled }}
+            - name: MTLS_ENABLED
```

Comment:
why does manager need MTLS env variable?  ot is a client

Reply:
when  mtls is not set the manager certificate is not needed in the tls handshake (only the server ca). When mutual tls authentication is used the manager certificate is not required.